docs/UserGuide.md

```diff
@@ -422,7 +420,11 @@ Format: `list -c`
 ### Filtering client profiles: `filter -c`
 
 Returns a filtered list of clients according to specified tags.
+
+TAG have to be fully specified (i.e. no partial tags like 'high' instead of 'high-end')
```

Comment:
am okay with this phrasing but not sure if it makes sense without the example of `high-end`

Reply:
Maybe it would be better if you phrase it as such?
'A TAG has to be **fully** and not partially typed out (i.e. 'high-end' instead of 'high')